plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/ServerEvaluationStrategy.java

```diff
@@ -61,10 +61,13 @@
      * @param internalAddress
      *         address passed into {@code getServers}; used as fallback if address cannot be
      *         retrieved from containerInfo.
+     * @param serverConfMap
+     *         additional Map of {@link ServerConfImpl}.
```

Comment:
It is unclear from javadocs why this additional map is needed. Please explain it in javadocs.